airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2134,6 +2134,7 @@ components:
       schema:
         type: integer
         minimum: 1
+        maximum: 100
```

Comment:
I have introduced a change that will facilitate the introduction of a limit that can be defined by the user. 
https://github.com/apache/airflow/pull/9405
In this way, we will not have a fixed value in API and users will be able to adjust this limit according to their requirements.

Reply:
- isn't this supposed to be done with custom validators? 

- isn't the intention to not allow the user to call the API with too high limit?

Reply:
If you want, you can do it using your method, but I think my idea seems simpler to do.

The intention is to limit, but the limit depends on the specific deployment. If your server has 64 GB RAM, then you can set a different limit than if your server has only 1 GB.